.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.0.1
```

Comment:
Hmm this doesn't seem right. Which version of pre-commit did you have installed @LucaMantani when you tested this?

Reply:
I have 4.0.1 locally

Reply:
Ok, all good then, let's merge